tests/acquia_lift.test

```diff
@@ -2519,6 +2520,45 @@ class AcquiaLiftWebTestCampaignWizard extends AcquiaLiftWebTestBase {
     $this->assertOptionSelected('edit-audiences-everyone-else-assignment', $first_option_set->options[0]['option_id']);
   }
 
+
+  /**
+   * Test doing a straight test, i.e. all variations assigned to "Everyone Else".
+   */
+  function testNoTargeting() {
```

Comment:
Is this test new?  I could have sworn there was something like this already.  It looks very similar to testTargeting().